frontend/src/pages/registration/RegistrationPageView.jsx

```diff
@@ -1,93 +1,126 @@
+                            onChange={(e) => setUsername(e.target.value)}
+                        />
 
-                    {error.userName && (
-                        <p className={classes.error}>{error.userName}</p>
-                    )}
-                </div>
+                        {error.userName && (
+                            <p className={classes.error}>{error.userName}</p>
+                        )}
+                    </div>
 
-                <div className={classes.email}>
-                    <InputField
-                        label="Email"
-                        type="email"
-                        name="email"
-                        value={value.email}
-                        onChange={onInputChange}
-                    />
+                    <div className={classes.input}>
+                        <TextField
+                            label="Your Email"
+                            type="email"
+                            variant="outlined"
+                            value={email}
+                            fullWidth
+                            onChange={(e) => setEmail(e.target.value)}
+                        />
 
-                    {error.email && (
-                        <p className={classes.error}>{error.email}</p>
-                    )}
-                </div>
-                <div className={classes.password}>
-                    <InputField
-                        label="Password"
-                        type="password"
-                        name="password"
-                        value={value.password}
-                        onChange={onInputChange}
-                    />
-                    {error.password && (
-                        <p className={classes.error}>{error.password}</p>
-                    )}
+                        {error.email && (
+                            <p className={classes.error}>{error.email}</p>
+                        )}
+                    </div>
+                    <div className={classes.input}>
+                        <FormControl variant="outlined" fullWidth>
+                            <InputLabel htmlFor="outlined-adornment-password">
+                                Password
+                            </InputLabel>
+                            <OutlinedInput
+                                id="outlined-adornment-password"
+                                type={showPassword ? 'text' : 'password'}
+                                value={password}
+                                onChange={(e) => setPassword(e.target.value)}
+                                endAdornment={
+                                    <InputAdornment position="end">
+                                        <IconButton
+                                            aria-label="toggle password visibility"
+                                            onClick={setShowPassword}
+                                            onMouseDown={setShowPassword}
```

Comment:
Needs some logic to set it to !showPassword, currently when they click it once they can't toggle it back off

Reply:
oh true, mb